source/jormungandr/jormungandr/utils.py

```diff
@@ -494,6 +494,30 @@ def get_pt_object_from_json(dict_pt_object, instance):
     return pt_object
 
 
+def transform_entrypoint(dict_pt_object, uri):
```

Comment:
I can't find a good name for this function... but I'd say we should try to find another name. Here are suggestions for brainstorming, maybe you will find other ideas?

Building the name of the function with this shape.
```<verb>_<name>```

`<verbs>`:
- improve
- customize
- replace
- rectify

`<names>`:
- entrypoint_detail
- address (`replace_address` for example)

other ideas:
- `rectify_entrypoint_with_uri`
- `entrypoint_uri_refocus` (I kind of like this one, we're changing back the entrypoint to refocus on the end-user information: the `from` or the `to`)

None of them are really good, but I'm hoping to help find ideas.

Reply:
Done